src/commandDetails/coin/check.ts

```diff
@@ -5,28 +5,39 @@ import {
   CodeyCommandOptionType,
   CodeyCommandResponseType,
   SapphireMessageExecuteType,
-  SapphireMessageResponse
+  SapphireMessageResponse,
+  getUserFromMessage,
 } from '../../codeyCommand';
 import { getCoinBalanceByUserId } from '../../components/coin';
 import { getCoinEmoji } from '../../components/emojis';
 
 // Check a user's balance
 const coinCheckExecuteCommand: SapphireMessageExecuteType = async (
   _client,
-  _messageFromUser,
+  messageFromUser,
   args
 ): Promise<SapphireMessageResponse> => {
-  // Mandatory argument is user
-  const user = <User>args['user'];
+  let user: User;
+  let displayMessage: string;
+
+  // use the caller as a default user if no argument is provided
+  if (args['user']) {
+    user = <User>args['user'];
+    displayMessage = `${user.username} has`;
+  } else {
+    user = getUserFromMessage(messageFromUser);
+    displayMessage = `You have`;
+  }
   // Get coin balance
   let balance: number;
   try {
     balance = await getCoinBalanceByUserId(user.id);
   } catch (e) {
     return `Could not fetch the user's balance, contact a mod for help`;
   }
+
   // Show coin balance
-  return `${user.username} has ${balance} Codey coins ${getCoinEmoji()}.`;
+  return `${displayMessage} ${balance} Codey coins ${getCoinEmoji()}.`;
 };
 
 export const coinCheckCommandDetails: CodeyCommandDetails = {
```

Comment:
If you're removing balance, make `aliases` also have `balance, bal and b`, since people use these commands regularly.

Reply:
Done